extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json

```diff
@@ -5,131 +5,178 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/experience/customerJourneyManagement/ajo",
+  "$id": "https://ns.adobe.com/xdm/ajo",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "title": "Adobe Profile - AJO Entities",
-  "description": "Denormalized AJO Entities.",
+  "title": "AJO Entity Fields",
+  "description": "A set of meta-data fields related to various AJO entities which are related to message sent to profile.",
   "type": "object",
   "meta:extensible": true,
   "meta:abstract": false,
-  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/ajo/entity"],
   "definitions": {
     "ajo": {
       "properties": {
-        "https://ns.adobe.com/experience/customerJourneyManagement/ajo/campaign": {
+        "https://ns.adobe.com/xdm/ajo/campaign": {
```

Comment:
Let's not use xdm here as we are not creating a standard XDM component here. Please use the same terminology as you used before like "https://ns.adobe.com/experience/customerJourneyManagement/ajo/campaign"

Please remember that property names would convert to hierarchies in the actual schema when the datasets stored. 
Example: this property would become 
_experience
  customerJourneyManagement
    ajo
       campaign (This is an object with all the other properties nested under this)

I would suggest to go through other XDM components defined in the customer journey management folder and create your properties accordingly.

Reply:
> _experience
> customerJourneyManagement

@mjasrotia Does `_experience` here has any bearing of this being an experience event data rather than record data. Or even for normal record data the recommendation is to use `_experience`

Reply:
Done.